docs/application/web/api/6.5/device_api/mobile/tizen/se.html

```diff
@@ -1322,6 +1326,48 @@ <h4>Methods</h4>
         </ul>
 </div>
 </dd>
+<dt class="method" id="SEChangeListener::onUSBCardInserted">
+<a class="backward-compatibility-anchor" name="::SecureElement::SEChangeListener::onUSBCardInserted"></a><code><b><span class="methodName">onUSBCardInserted</span></b></code>
+</dt>
+<dd>
+<div class="brief">
+ Called when a USB Secure Element reader is inserted.
+            </div>
+<div class="synopsis"><pre class="signature prettyprint">void onUSBCardInserted(<a href="#Reader">Reader</a> reader);</pre></div>
+<p><span class="version">Since: </span>
+ 6.5
+            </p>
+<div class="parameters">
+<p><span class="param">Parameters:</span></p>
+<ul>
+          <li class="param">
+<span class="name">reader</span>:
+ An inserted USB Secure Element reader.
+                </li>
+        </ul>
+</div>
+</dd>
+<dt class="method" id="SEChangeListener::onUSBCardRemoved">
+<a class="backward-compatibility-anchor" name="::SecureElement::SEChangeListener::onUSBCardRemoved"></a><code><b><span class="methodName">onUSBCardRemoved</span></b></code>
+</dt>
+<dd>
+<div class="brief">
+ Called when a USB Secure Element reader is removed.
+            </div>
+<div class="synopsis"><pre class="signature prettyprint">void onUSBCardRemoved(<a href="#Reader">Reader</a> reader);</pre></div>
+<p><span class="version">Since: </span>
+ 6.5
+            </p>
+<div class="parameters">
+<p><span class="param">Parameters:</span></p>
+<ul>
+          <li class="param">
+<span class="name">reader</span>:
+ A removed USB Secure Element reader.
```

Comment:
USB Secure Element reader is removed.

Reply:
This sentence is correct - `reader` is an object, that represents the USB Secure Element reader that is inserted.